drivers/Makefile.dep

```diff
@@ -36,6 +36,15 @@ ifneq (,$(filter at,$(USEMODULE)))
   USEMODULE += isrpipe_read_timeout
 endif
 
+ifneq (,$(filter at24cxxx,$(USEMODULE)))
+  FEATURES_REQUIRED += periph_i2c
+  USEMODULE += xtimer
+endif
+
+ifneq (,$(filter at24cxxx_eepreg_wrapper,$(USEMODULE)))
```

Comment:
I think that `at24cxx_eeprom_wrapper` would be a better name, as every user of the [EEPROM interface](http://api.riot-os.org/group__drivers__periph__eeprom.html) could use AT24CXXX EEPROM using that wrapper, not only the EEPROM registry.

Reply:
ok, gonna rename it